packages/v4/patternfly-docs/content/design-guidelines/demos/primary-detail/primary-detail.md

```diff
@@ -27,39 +30,40 @@ When the page content has a white background, the drawer can be placed on the sa
+
+### Responsive behavior
+The mobile behavior of the interaction will generally mirror the desktop behavior. Mobile views will only show 1 component at a time, either the primary component or the detail component. Upon selecting an item from the primary component, the detail component will slide in from the right and will overlay the primary component, taking the whole width of the viewport. The primary component’s title will remain constant as the header with clear navigation back to the primary component. The detail component will also be dismissable and will slide out to the right upon closing, revealing just the primary component. The viewport will switch to mobile view at a defined breakpoint, usually when it drops below 2x (which is the typical mobile width). 
  
-In tablet/dt screens, the primary component may switch to its mobile view—while still maintaining a side-by-side view with the detail drawer—earlier than the typical mobile breakpoint. 
+<img src="./img/mobile-primary-detail-in-card-view.png"  alt="Image showing primary-detail component mobile behavior in card view"  width="943"/> 
  
-<img src="./img/side-by-side-mobile.png"  alt="primary-detail with side-by-side view on mobile"  width="375"/> 
+In cases where the detail component in desktop view is non-dismissible and active at all times, the detail component becomes dismissible when switching to mobile view, and behaves as described above.
  
-If the primary and detail components are the same width or take up the same portion of the screen, then both will resize until they reach the mobile view’s defined breakpoint.
+<img src="./img/mobile-primary-detail-in-list-view.png"  alt="Image showing primary-detail component mobile behavior in non-dismissable list view"  width="2520"/> 
  
-If the primary and detail components are different sizes, the larger container will resize, and the smaller container will remain as a fixed size until they reach the device’s defined breakpoint.
+## Variations
  
-## Selecting rows
+### Selecting rows
 Row selection differs from bulk/checkbox selection, and both selection types operate independent of each other.
  
 When checkbox select and row select are both available in a primary-detail layout, the details pane will populate an empty state and guide users to either make an action in the toolbar or clear the selected checkboxes to return to the selected row’s detail view. This empty state should show to alleviate confusion for the different select states when both are present in a table.
  
-<img src="./img/primary-detail-bulk-selection.png"  alt="primary-detail with two items selected"  width="990"/> 
+<img src="./img/primary-detail-multi-select.png"  alt="Image showing primary-detail component with multi-select items"  width="1500"/> 
```

Comment:
The body text of the empty state in this image should probably be center aligned instead of left aligned to match empty state styling!